docs/develop/persisting-add-in-state-and-settings.md

```diff
@@ -17,8 +17,10 @@ To do that, you can:
   - Custom XML stored in the document.
 
 - Use techniques provided by the underlying browser control: browser cookies, or HTML5 web storage ([localStorage](https://developer.mozilla.org/docs/Web/API/Window/localStorage) or [sessionStorage](https://developer.mozilla.org/docs/Web/API/Window/sessionStorage)).
+    > [!NOTE]
+    > The user can block browser-based storage techniques depending on the settings they choose.
 
-This article focuses on how to use the Office JavaScript API to persist add-in state. For examples of using browser cookies and web storage, see the [Excel-Add-in-JavaScript-PersistCustomSettings](https://github.com/OfficeDev/Excel-Add-in-JavaScript-PersistCustomSettings).
```

Comment:
Let's keep the link to the sample. Even though it is archived. It is still useful.

Reply:
thanks, done!